src/redux/actionCreators.js

```diff
@@ -86,6 +86,7 @@ export function run(src) {
 }
 
 export function changeSource(source) {
+  localStorage.setItem('source', source);
```

Comment:
put this 'source' string in a constant and give it a more unique name since it is pretty much a global